app/pods/bucket-type/model.js

```diff
@@ -28,6 +28,23 @@ var BucketType = DS.Model.extend({
     bucketList: DS.belongsTo('bucket-list'),
 
     /**
+     * Bucket Type Properties object.
```

Comment:
So, out of sheer curiosity, why change the order of the properties? It was in alphabetical order before - what's the logic now? :)